src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -1074,7 +1115,6 @@ initialize_query_table()
   emplace_sysfs_get<query::xocl_errors>                        ("", "xocl_errors");
 
   emplace_func0_request<query::pcie_bdf,                       bdf>();
-  emplace_func0_request<query::kds_cu_info,                    kds_cu_info>();
   emplace_func0_request<query::instance,                       instance>();
```

Comment:
removed duplicate line of line #1071
emplace_func0_request<query::kds_cu_info,                    kds_cu_info>();